### ISSUE TITLE
Face ID license and redist of the lora version

### ISSUE THREAD

**Issue by cubiq** [tencent-ailab/IP-Adapter#188]
I was going to put a lora version of the FaceID on hugging face but I noticed that the license is not-commercial for research only. Not sure if it's okay to redistribute the converted model, would you add maybe my model to your repo?

**Comment by xiaohu2015** [comment]
the best way is you make a PR in my huggingface repo and I will merge it. Also you can upload into your repo, but you should specify non-commercial use in your repo.

**Comment by whiterose199187** [comment]
hello @xiaohu2015 

May I ask why the license doesn't permit commercial use? Is it because using insightface? Would the same restrictions apply if one was to train the model themselves? I have a client who is interested in deploying this commercially, would it be possible for them to license this for commercial use?

This super cool btw, thanks again for your project! :)

**Comment by xiaohu2015** [comment]
@whiterose199187 yes, beacuse we use ininsightface to train the model, hence the trained model is not allowed for commercial use. But IP-Adapter code is open (with apache 2.0 license), hence you can use it to train your commercial model.

**Comment by whiterose199187** [comment]
Got it. Thanks!

**Comment by Shandelier** [comment]
@xiaohu2015 does the insightface is used for creating image embbeding or for processing of training data?

**Comment by Blu-Stat-ic** [comment]
Hi @xiaohu2015,

I'm George Gomes from Static Productions. I've been following your IP-Adapter FaceID model and appreciate the work you've done on it.

As mentioned in Issue #188 that the non-commercial license is due to the InsightFace dependency (not the code itself), we're interested in pursuing commercial licensing for this model for our production workflow.

**Our Use Case:**
We use IP-Adapter FaceID for face-swapping and identity preservation in video production workflows as part of Static Productions' content creation pipeline. This involves professional-grade applications including:
- Commercial video editing and post-production
- Content creation for multiple clients
- Integration into automated production pipelines

**Questions for Licensing:**
1. What are the commercial licensing terms available? (One-time fee, annual license, or usage-based?)
2. Are there different tiers based on revenue scale or number of users?
3. What documentation/agreements are required to obtain a commercial license?
4. Is attribution still required even with a commercial license?
5. What is the typical timeline for license approval and payment processing?

We're prepared to move quickly on this and would appreciate your guidance on the process. Please let us know what information you need from us to proceed.

Looking forward to your response.

Best regards,
George Gomes
Static Productions

**Comment by Blu-Stat-ic** [comment]
Hi @xiaohu2015,

Thank you for the response. To move forward with our commercial licensing request, could you please provide more specific details:

**Licensing Process Details:**
1. What is the exact process to obtain a commercial license? (Email application, form submission, etc.)
2. Are there any fees associated with the commercial license? If yes, what are the fee structures?
3. Do we need to sign a formal agreement or provide additional documentation?
4. Is there a specific contact person for licensing inquiries beyond this issue thread?

**Timeline & Next Steps:**
5. What is the typical timeline from application to receiving the commercial license?
6. Are there any restrictions on how many models can be trained using IP-Adapter FaceID under a commercial license?

We're ready to proceed as soon as you provide these details. Please let us know if you need any additional information from our end.

Best regards,
George Gomes
Static Productions

**Comment by Blu-Stat-ic** [comment]
Thank you for your response. Could you please provide more specific details about the commercial licensing process?

**Licensing Process Details:**
1. What is the exact process to obtain a commercial license? (Email application, form submission, etc.)
2. Are there any fees associated with the commercial license? If yes, what are the fee structures?
3. Do we need to sign a formal agreement or provide additional documentation?
4. Is there a specific contact person for licensing inquiries beyond this issue thread?

**Timeline & Next Steps:**
5. What is the typical timeline from application to receiving the commercial license?
6. Are there any restrictions on how many models can be trained using IP-Adapter FaceID under a commercial license?

We're ready to proceed as soon as you provide these details. Please let us know if you need any additional information from our end.

Best regards,
George Gomes
Static Productions

**Comment by Blu-Stat-ic** [comment]
Hi @xiaohu2015,

Following up on our commercial licensing inquiry for IP-Adapter FaceID. We posted detailed questions about the licensing process and fee structure on 2026-09-03 (comments #5522772654 and #5523048874).

Could you please provide guidance on:
1. The commercial licensing application process
2. Fee structures for commercial use
3. Required documentation/agreements
4. Timeline for license approval

We're ready to proceed with the full licensing process as soon as we receive your response. This is critical for our production workflow involving face-swapping and identity preservation in video production.

Thank you for your time and assistance.

Best regards,
George Gomes
Static Productions
